cornice/tests/validationapp.py

```diff
@@ -62,11 +62,8 @@ def get2(request):
 service3 = Service(name="service3", path="/service3")
 
 
-def _accept(request):
-    return ('application/json', 'text/json')
-
-
-@service3.get(accept=_accept)
+@service3.get(accept=lambda request: ('application/json', 'text/json'))
+@service3.put(accept=lambda request: 'text/json')
```

Comment:
might be better to define another service (and name it properly, we shouldn't continue with this bad habit of naming these "serviceX" !)